gnomad/sample_qc/ancestry.py

```diff
@@ -50,6 +50,7 @@
     "oeu": "Other European",
     "onf": "Other Non-Finnish European",
     "unk": "Unknown",
+    "remaining": "Remaining",
```

Comment:
This is resulting in a python error when merging into main, because it was added separately in that branch, on a different line

https://github.com/broadinstitute/gnomad_methods/blob/main/gnomad/sample_qc/ancestry.py#L35